python/cuvs/cuvs/neighbors/brute_force/brute_force.pyx

```diff
@@ -163,11 +168,15 @@ def search(Index index,
     >>> # Search using the built index
     >>> queries = cp.random.random_sample((n_queries, n_features),
     ...                                   dtype=cp.float32)
+    >>> # Build prefilters
+    >>> n_bitmap = np.ceil(n_samples * n_features / 32).astype(int)
+    >>> bitmap = cp.random.random_sample((n_bitmap), dtype=cp.uint32)
```

Comment:
This line is throwing an exception in the doctests in CI:

```
TypeError: cupy.random only supports float32 and float64
```

(You can run this locally with w/ `pytest python/cuvs/cuvs/test/test_doctests.py` )

Reply:
Oh! Sorry for the low-level mistake. I did not pay enough attention to the example in a comment; thank you for pointing it out!